### PR TITLE
Add lxqt-panel

### PR DESCRIPTION
**Pull request by Besanon**
Panel for lxqt desktop group

### CODE REVIEW

File: apparmor.d/groups/lxqt/lxqt-panel [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  @{exec_path} mr,
+
+  @{bin}/exo-open rix,
+  @{bin}/nm-applet rPx,
+  @{bin}/nm-connection-editor rPx,
+  @{bin}/xdg-open rPx,
+  @{bin}/ControlPanel rPx,
+
+  /usr/lib{,32,64}/lxqt-panel/*.so mr, # LXQT-Plugins
+  /usr/lib{,32,64}/lxqt-config/*.so mr, # LXQT-Plugins
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-panel:35]: Use `@{lib}` instead of `/usr/lib{,32,64}`

File: apparmor.d/groups/lxqt/lxqt-panel [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+  owner @{user_config_dirs}/lxqt/#* rw,
+  owner @{user_config_dirs}/lxqt/panel.conf rw,
+  owner @{user_config_dirs}/lxqt/panel.conf.lock rwk,
+  owner @{user_config_dirs}/lxqt/panel.conf.@{rand6} rw,
+  owner @{user_config_dirs}/lxqt/panel.conf.@{rand6} l -> @{user_config_dirs}/lxqt/#*,
+  owner @{user_config_dirs}/pulse/{,**} rwk,
+  owner @{user_config_dirs}/ibus/bus/{,**} rw,
+
+  @{run}/udev/data/* r,
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-panel:69]: That need to be restricted. Take a look at how it is used in the profiles. You usually only need to copy/paste existing rule from other profiles.

File: apparmor.d/groups/lxqt/lxqt-panel [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+  /var/lib/dbus/machine-id r,
+
+  owner @{HOME}/.config/menus/*.menu rw,
+  owner @{HOME}/.config/menus/applications-merged/ r,
+  owner @{HOME}/Desktop/*.desktop rw,
+  owner @{HOME}/Desktop/#@{int} rw,
+  owner @{HOME}/Desktop/*.desktop l -> @{HOME}/Desktop/#@{int},
+  owner @{HOME}/.local/share/desktop-directories/*.directory r,
+  owner @{HOME}/.local/share/gvfs-metadata/{,*} r,
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-panel:59]: You need to use the tunables such as `@{user_config_dirs}` cf https://apparmor.pujol.io/variables/

File: apparmor.d/groups/lxqt/lxqt-panel [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+  owner @{user_config_dirs}/lxqt/#* rw,
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-panel:61]: That should be: `@{user_config_dirs}/lxqt/#@{int}` (same elsewhere)

File: apparmor.d/groups/lxqt/lxqt-panel [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+  @{bin}/xdg-open rPx,
+  @{bin}/ControlPanel rPx,
+
+  /usr/lib{,32,64}/lxqt-panel/*.so mr, # LXQT-Plugins
+  /usr/lib{,32,64}/lxqt-config/*.so mr, # LXQT-Plugins
+
+  /usr/share/lxqt/helpers/*.desktop r,
+  /usr/share/lxqt/panel/plugins/{,*.desktop} r,
+  /usr/share/desktop-directories/{,**} r,
+  /usr/share/X11/locale/locale.alias r,
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-panel:40]: `/usr/share/X11/` is part of the `lxqt` abstraction.

File: apparmor.d/groups/lxqt/lxqt-panel [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+  @{bin}/nm-applet rPx,
+  @{bin}/nm-connection-editor rPx,
+  @{bin}/xdg-open rPx,
+  @{bin}/ControlPanel rPx,
+
+  /usr/lib{,32,64}/lxqt-panel/*.so mr, # LXQT-Plugins
+  /usr/lib{,32,64}/lxqt-config/*.so mr, # LXQT-Plugins
+
+  /usr/share/lxqt/helpers/*.desktop r,
+  /usr/share/lxqt/panel/plugins/{,*.desktop} r,
```
Comment (roddhjav) [inline apparmor.d/groups/lxqt/lxqt-panel:38]: Simplify as `/usr/share/lxqt/{,**}`

**Comment by roddhjav** [comment]
There is an issue between: 
- `abstractions/app-launcher-user`
- `@{bin}/exo-open rix,`
- `@{bin}/xdg-open rPx,`

Both `exo` & `xdg` are part of `@{open_path}`. They should be handled the same way.

See: https://apparmor.pujol.io/development/internal/#open-resources for more.

**Comment by Besanon** [comment]
what crashed the first test?

**Comment by Besanon** [comment]
ok, the new altered version is working without complaints in Arch linux

**Comment by roddhjav** [comment]
Thanks, it looks good now. Merged!

**Pull request merged**